modules/localization/ndt/ndt_localization.cc

```diff
@@ -254,7 +254,7 @@ bool NDTLocalization::IsServiceStarted() { return is_service_started_; }
 
 void NDTLocalization::FillLocalizationMsgHeader(
     LocalizationEstimate* localization) {
-  DCHECK_NOTNULL(localization);
+  CHECK_NOTNULL(localization);
```

Comment:
1. Use CHECK.
2. IMO, we can remove these CHECK on output pointers as it's pretty common sense.

Reply:
Yes, you are right for output pointers check, I will remove it.